### ISSUE TITLE
Plants dry out extremely fast

### ISSUE THREAD

**Issue by Multipogopuschel** [Regalis11/Barotrauma#7902]
- [ x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Without being affected by salt water, plants dry out way faster without watering them after last patch.
Plants lose over half of their water in ~10 minutes, while it took ~1-2 hours to lose them that much before the patch.
At the same time watering plants got way faster, too.

**Steps To Reproduce**
Use an older multiplayer campaign savegame.
Have already planted plants from before the patch in the savegame. 
Update the game and load the save game. Watering and drying out should be way faster now.

**Version**
Barotrauma v0.15.22.1 (ReleaseWindows, branch release, revision 898f64d2c0)
Windows 10

**Additional information**
We are playing a multiplayer campaign and I tried to fix the problem by replacing planters and planting new plants, but they still dry out as fast as the old plants.

**Comment by gerakis** [comment]
There is no way it took 2 hours. It was like 30 minutes at most before the patch. I dont know about now.

**Comment by Multipogopuschel** [comment]
I had sessions especially at the beginning of playing where I didn't water the plants at all (because I didn't know how at first) and they still lived. We play around 2-3 hour in a session, so it took extremely long for them to dry out. This is of course just an estimate, but even if it was 30 minutes before, it shouldn't be 10 minutes now, right? I can try to set a timer the next time I play, but nevertheless it is way faster now and I can hardly do any outside missions because of it.

**Comment by gerakis** [comment]
Im reading the source code at Barotrauma/Barotrauma/BarotraumaShared/SharedSource/Items/Components/Growable.cs /

There were changes but nothing jumps out saying they are dying faster. They added some unused stuff for fire damage, and a multiplier that slows down water consumption while fertilized.

**Comment by Oukido** [comment]
I have recently begun a new singleplayer campain with an overhauled version of a ship I made before. When I was testing the old save for comparison reasons I was able to observe insane activity of the plants in that save file. I had six planter boxes which produced about a full cargo crate every 10-15 minutes while also demanding frequent watering.
In the new save the plants are way slower in both production and also water consumption.
My guess here is that since the last time the old save was actively played something in the code changed regarding the fertilization. So the bug seems to be found in the old save game itself rather in the actual code, which can be expected in a wip where everything is subject to change.
